streams/catalog/src/main/java/com/hortonworks/streamline/streams/catalog/service/StreamCatalogService.java

```diff
@@ -2411,7 +2361,7 @@ public UDF addUDF(UDF udf) {
         return udf;
     }
 
-    public Map<String, Class<?>> loadUdfsFromJar(java.io.File jarFile) throws IOException {
+    public static Map<String, Class<?>> loadUdfsFromJar(java.io.File jarFile) throws IOException {
```

Comment:
I know this doesn't require any fields in StreamCatalogService, but if we would want to make it static, maybe better to have utility class instead. Just a 2 cents.